app/build.gradle

```diff
@@ -156,6 +156,13 @@ dependencies {
     // In case 'lifecycleAutotracking' is enabled
     implementation "androidx.lifecycle:lifecycle-extensions:$versions.snowplow_lifecycle"
 
+    //Crypto
+    implementation "androidx.security:security-crypto:$versions.crypto"
+
+    //SqlCipher
+    implementation "net.zetetic:android-database-sqlcipher:$versions.sqlcipher"
+    implementation "androidx.sqlite:sqlite:$versions.sqlite"
```

Comment:
change androidx.sqlite:sqlite to androidx.sqlite:sqlite-ktx

Reply:
Completed